wrappers/python/examples/roberta_training.py

```diff
@@ -0,0 +1,313 @@
+    num_hidden_layers=model_torch.config.num_hidden_layers,
+    num_attention_heads=model_torch.config.num_attention_heads,
+    intermediate_size=model_torch.config.intermediate_size,
+    intermediate_size_tile=args.intermediate_size_tile,
+    n_head_tile=args.n_head_tile,
+    dtype=args.dtype,
+    # flash_attention=args.flash_attention
+)
+
+print(bert_config_nntile)
+
+bert_nntile, next_tag = BertForMaskedLM_nntile.from_torch(model_torch,
+                                                args.minibatch_size,
+                                                args.minibatch_size_tile,
+                                                args.seq_len,
+                                                args.seq_len_tile,
+                                                bert_config_nntile,
+                                                next_tag)
+time1 = time.time() - time0
+print("Converting PyTorch model to NNTile",
+        "requires {} seconds".format(time1))
+del model_torch
+
+splitted_datasetfile = args.dataset_file.split("/")
+if splitted_datasetfile[-1] == "train.bin":
+    train_data = np.memmap(Path(args.dataset_path) /
+                            args.dataset_file,
+                            dtype=np.uint16, mode='r')
+    train_tokens_raw = np.array(train_data, order='F', dtype=np.int64)
+    del train_data
+else:
+    raise ValueError("Only train.bin file is accepted"
+                        " for training dataset!")
+
+num_train_tokens = train_tokens_raw.shape[0]
+
+num_train_seq = num_train_tokens // (args.seq_len + 1)
+num_train_batches = num_train_seq // args.batch_size
+num_train_tokens_truncated = num_train_batches * (args.batch_size
+        * (args.seq_len + 1))
+train_tokens_trunc = np.array(
+    train_tokens_raw[:num_train_tokens_truncated],
+    order='F', dtype=np.int64)
+train_tokens = train_tokens_trunc.reshape(num_train_batches,
+                                    num_minibatch,
+                                    args.minibatch_size,
+                                    args.seq_len + 1)
+
+time0 = time.time()
+batch_masked_data = []
+batch_labels = []
+x_traits = nntile.tensor.TensorTraits(
+        [args.seq_len, args.minibatch_size],
+        [args.seq_len_tile, args.minibatch_size_tile])
+x_distr = [0] * x_traits.grid.nelems
+
+rng = np.random.default_rng()
+for epoch_idx in range(args.nepochs):
+    masked_data_epoch = []
+    labels_epoch = []
+    for i in range(num_train_batches):
+        minibatch_masked_data = []
+        minibatch_labels = []
+        for j in range(num_minibatch):
+            current_mask = np.zeros((args.minibatch_size,
+                                     args.seq_len), dtype=bool)
+            idx_masked_tokens = rng.choice(args.seq_len,
+                                           size=(args.minibatch_size,
+                                                 args.n_masked_tokens_per_seq))
+            current_mask[:, idx_masked_tokens] = 1
```

Comment:
mask shall be different for all sequences in a batch, i think. Not a single mask for entire batch.